LICENSE

```diff
@@ -1,20 +1,22 @@
 The MIT License (MIT)
 
+Copyright (c) 2020 Blaine Bublitz <blaine.bublitz@gmail.com>, Eric Schoffstall <yo@contra.io> and other contributors
 Copyright (c) 2014 Artem Medeusheyev
```

Comment:
If these aren't somehow on the same line, the GitHub UI displays that we don't have a license. I've usually done something like:
```suggestion
Copyright (c) 2014 Artem Medeusheyev & 2020 Blaine Bublitz <blaine.bublitz@gmail.com>, Eric Schoffstall <yo@contra.io> and other contributors
```